ui/store/actions.ts

```diff
@@ -2175,6 +2175,18 @@ export function createCancelTransaction(
         [txId, customGasSettings, { ...options, actionId }],
         (err, newState) => {
           if (err) {
+            if (
```

Comment:
As this is distinct logic that can be tested, is it worth a quick unit test given it's an error scenario and the UI is historically lacking in coverage?

Reply:
This is added in https://github.com/MetaMask/metamask-extension/pull/22943/commits/307b980cae1fc3194981d7d93f260ef011cfe09c